zanata-server/runapp.sh

```diff
@@ -6,11 +6,16 @@ if [ -n "${ZANATA_VERSION-}" ];then
     ZanataVer=$ZANATA_VERSION
 else
     ## Use the ZANATA_VERSION from Dockerfile, line "ARG ZANATA_VERSION="
-    ZanataVer=$(sed -r -n -e '/^ARG\s+ZANATA_VERSION/ s/^ARG\s+ZANATA_VERSION\s*=\s*(.*)\s*/\1/p' < $ScriptDir/Dockerfile)
+    ZanataVer=latest
```

Comment:
The only problem it that if we forget to push the version, it won't have error message. Perhaps we can change comment into 
"Use the latest version (not necessary the version defined in Dockerfile"